Blockzilla/BrowserViewController.swift

```diff
@@ -1035,6 +1032,19 @@ extension BrowserViewController: PhotonActionSheetTransitionDelegate {
     func photonActionSheetDidDismiss() {
         darkView.isHidden = true
     }
+    func photonActionSheetDidToggleProtection(enabled: Bool) {
+        if enabled {
```

Comment:
nit: how do you feel about turning this into a ternary operator:

```enabled ? webViewController.enableTrackingProtection() : webViewController.disableTrackingProtection()```

IMO it just cleans things up a little bit, but if you disagree let me know :)